### ISSUE TITLE
PCB production

### ISSUE THREAD

**Issue by mevlad24** [lal12/vaillant-heater-control-esp#1]
Hello, 
please are you aware of anyone producing the PCB you are describing on your GH page?
Thank you
 Vladimir

**Comment by lal12** [comment]
No. I only ordered a few for myself. However I had issues getting it to work anyway. The temperature controlling part works fine. But the serial part didn't work properly. I am not exactly sure what's the issue, probably the heater mcu doesn't provide enough current to drive the optoptocoupler at least while maintaining a clean signal.

I also had issues with other means, e.g. only some serial to USB converters worked and even the one which worked stopped after a while, the constant connection (and polling) might have broke the IOs in the heater. But I never really looked into that after more closely it stopped working.

**Comment by LordTompa** [comment]
Hi, I built the Paltine a year ago.
Only the part that is needed to control the temperature, but that's all I wanted. 
Everything worked perfectly. 
I ordered the boards from Jlcpcb and still have 4 boards left. 
If you are from Germany, it is no problem for me to send you 1-2 boards.
But as a note, the boards are unassembled and I'm pretty sure I don't have the required components anymore.
So you would have to get and solder them yourself
